### PR TITLE
Track sequential steps as virtual page views

### PR DESCRIPTION
**Pull request by adamlofting**

### CODE REVIEW

File: pages/sequential.jsx [hunk 1]
```diff
@@ -4,6 +4,8 @@ import Header from '../components/header.jsx';
 import Link from '../components/link.jsx';
 import {IntlMixin} from 'react-intl';
 
+var ga = require('react-ga');
```
Comment (adamlofting) [inline pages/sequential.jsx]: Should I be using `import` like above? This looks like shiny new Javascript I'm not yet familiar with :stuck_out_tongue_closed_eyes:

**Comment by adamlofting** [comment]
@ScottDowne R?

(I can confirm data is arriving in GA via this running locally)

File: pages/sequential.jsx (hunk 1 above)
Comment (alicoding) [inline pages/sequential.jsx]: Not require really :P we're just trying to be cool haha

**Comment by alicoding** [comment]
R+

**Comment by adamlofting** [comment]
Completes more of #16

**Pull request merged**